### DISCUSSION TITLE
Can Netdata be installed and enjoyed on a closed environment without internet access?

### DISCUSSION THREAD

**Discussion by mrbillr01** [netdata/netdata#14526] [Q&A]
Would like to use Netdata on our secure air-gapped network with no outside access.

**Comment by thiagoftsm** [comment]
Hello @mrbillr01 ,

Yes, you can install netdata on off-line environments, please, take a look in our [documentation](https://learn.netdata.cloud/docs/agent/packaging/installer/methods/offline), any doubt let us know.

Best regards
